src/core/elements/blockquotes.scss

```
@@ -10,5 +10,13 @@
     padding-block: var(--blockquote-padding-y, var(--space-s));
     padding-inline: var(--blockquote-padding-x, var(--space-s));
     position: relative;
+    &:before {
```

I think this adds styles to the default `blockquote` element, if we wanna apply it with the `.quote` modifier class, we could add it like:

```suggestion
    &:is(.quote):before {
```
or just:
```suggestion
 &.quote:before {
```